example/dags/postgres_sample_dag.py

```diff
@@ -74,8 +74,8 @@ def connection_string():
 
 def create_table_extract_job(**kwargs):
     where_clause_suffix = textwrap.dedent("""
-        where table_schema in {schemas}
-    """)
+        where table_name in {schemas}
```

Comment:
why not table_schema in {schemas} but table_name?

Reply:
Hello, it seems that I changed to `table_name` for my use case. I also saw that there is an unused `OPTIONAL_TABLE_NAMES` in the code. Perhaps we can change the suffix such that, if `OPTIONAL_TABLE_NAMES` is not empty, the suffix will have an `AND table_name IN (...)`?

Reply:
Hi, I changed back to `table_schema`. Thought the other design considerations should be reserved for other pull requests :)

Reply:
thanks @darensin01 :) lgtm